### ISSUE TITLE
do you have a Laravel Example?

### ISSUE THREAD

**Issue by krasho** [arquivei/laravel-prometheus-exporter#17]
Hello. I am trying to implement the package in a laravel 8 project, do you have an example? I think the documentation an example is to Lumen.

thanks

**Comment by ALTELMA** [comment]
Yes. That's true and the document needs to update. Because the middleware in the case of Lumen not existing too.
